.env.sample

```diff
@@ -1,5 +1,6 @@
 ADMIN=NAME,email@host.ca
 ALLOWED_HOSTS=rebootcanadadb.herokuapp.com,rebootcanada.herokuapp.com,localhost,0.0.0.0,127.0.0.1
+CELERY_RESULT_BACKEND="rpc://"
```

Comment:
ah interesting. looks like `amqp` was deprecated. JFYI prod is still using amqp backend for results..
https://stackoverflow.com/a/48921731